tests/sqllogictests/suites/base/01_system/01_0002_system_query_log.test

```diff
@@ -42,6 +42,27 @@ select count(*) from tbl_01_0002 where a > 1
 # ----
 # 1
 
+statement ok
+select * from tbl_01_0002 where a=1;
```

Comment:
We should add a query(not pretty format) like:
```
select * from tbl_01_0002   where a = 1;
```
To make sure the query_hash are same.

Reply:
Added